confd/etc/calico/confd/templates/bird6_ipam.cfg.template

```diff
@@ -47,5 +47,13 @@ filter calico_kernel_programming {
   {{- end}}
 
 {{- end}}
+{{range ls "/v1/ipam/v6/pool"}}{{$data := json (getv (printf "/v1/ipam/v6/pool/%s" .))}}
```

Comment:
@caseydavenport the IPv4 counterpart of this contains a line checking if `network_v4` exists (IPIP needs this info). Would this be necessary for IPv6? I tried leaving it in but had a bit of trouble, is some format different in this case?

Reply:
I don't think that's necessary for IPv6 - it's only used for cross subnet checks in IPIP, but we don't support IPIP for v6.